server/views/pages/recategoriserHome.html

```diff
@@ -64,7 +65,14 @@ <h1 class="govuk-heading-l">Category reviews for prisoners</h1>
           {{ row.nextReviewDateDisplay }}
         {% endif %}
       </td>
-      <td class="govuk-table__cell"><a target="_blank" href="{{ row.offenderNo | offenderLink }}">{{ row.displayName }}</a></td>
+      <td class="govuk-table__cell"><a target="_blank" href="{{ row.offenderNo | offenderLink }}">{{ row.displayName }}</a>
+        {% if row.securityReferred %}
+        {{ govukTag({
+        text: 'Security Referred',
```

Comment:
Happy to change this text if we think there's something better?

Reply:
That looks good, but maybe some hover text or similar would be good too , e.g. 'The security team have arranged that they will automatically be referred to security when their next category review is started'. Possibly should involve Martin as Im not sure hover text is very 'accessible'.

Reply:
Also As this can apply to initial cats too, we need a label on the initial todo list as well